Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using Aquality.WinAppDriver.Applications;
+using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Tests.Applications.Locators;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Aquality.WinAppDriver.Tests.Elements
+{
+    public class ElementTests : TestWithApplication
+    {
+        private readonly ITextBox rightArgumentTextBox = new CalculatorWindow().RightArgumentTextBox;
+        private const string ExpectedValue = "2";
+        private static readonly By ElementLocator = By.XPath("//*[@id='111111']");
+        const string ElementDescription = "Not present element";
+
+        private IElementFactory Factory => ApplicationManager.GetRequiredService<IElementFactory>();
+
+        [Test]
+        public void Should_SendKeys()
+        {
+            rightArgumentTextBox.SendKeys(ExpectedValue);
+            Assert.AreEqual(ExpectedValue, rightArgumentTextBox.Value);
+        }
+
+        [Test]
+        public void Should_ThrowException_InSendKeys_WhenNullIsSend()
+        {
+            Assert.Throws<ArgumentNullException>(() => rightArgumentTextBox.SendKeys(null));
+        }
+
+        [Test]
+        public void Should_GetNullValue_InSendKeys_WhenEmptyStringIsEntered()
+        {
+            rightArgumentTextBox.SendKeys(string.Empty);
+            Assert.IsNull(rightArgumentTextBox.Value);
+        }
+
+        [Test]
+        public void Should_SetCorrectLocatorInConstructor()
+        {
+            Assert.AreEqual(ElementLocator, GetLabel().Locator);
+        }
+
+        [Test]
+        public void Should_SetCorrectNameInConstructor()
+        {
+            Assert.AreEqual(ElementDescription, GetLabel().Name);
+        }
+
+        [Test]
+        public void Should_GetCorrectText_FromEmptyField()
+        {
+            Assert.AreEqual(new CalculatorWindow().ResultsLabel.Text, string.Empty);
+        }
+
+        [Test]
+        public void Should_GetElement_ElementIsPresent()
+        {
+            Assert.NotNull(rightArgumentTextBox.GetElement());
+        }
+
+        [Test]
+        public void Should_ThrowNoSuchElementException_InGetElement_WhenElementIsNotPresent()
+        {
+            Assert.Throws<NoSuchElementException>(() => GetLabel().GetElement(TimeSpan.Zero));
+        }
+
+        private IElement GetLabel() => Factory.GetLabel(ElementLocator, ElementDescription);
```

Comment:
looks like a property, please, convert to property and put it among other properties upper

Reply:
done